qiskit_bot/community.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+MONITORED_REPOS = ['qiskit-terra']
+EXCLUDED_USER_TYPES = ['Bot', 'Organization']
```

Comment:
This should be in the config file level instead of hard coding it here. That way on the deployment side if we want to add more repos it's a simple config change and doesn't require us to update the code. For monitored repos we should have an option here: https://github.com/Qiskit/qiskit-bot/blob/master/qiskit_bot/config.py#L46 like `uses_community_label`. 

For excluded user types we can probably leave this hard coded because I doubt this will change often, but if you wanted to adding a top level optional list would work too.